modules/core/include/opencv2/core/hal/intrin_neon.hpp

```diff
@@ -1203,17 +1203,43 @@ inline void v_store_low(_Tp* ptr, const _Tpvec& a) \
 inline void v_store_high(_Tp* ptr, const _Tpvec& a) \
 { vst1_##suffix(ptr, vget_high_##suffix(a.val)); }
 
+#define OPENCV_HAL_IMPL_NEON_WORKAROUND_64(_Tpvec, _Tp, suffix) \
+inline _Tpvec v_load(const _Tp* ptr) \
+{ return _Tpvec(vreinterpretq_##suffix##_u8(vld1q_u8((const unsigned char*)ptr))); } \
+inline _Tpvec v_load_aligned(const _Tp* ptr) \
+{ return _Tpvec(vreinterpretq_##suffix##_u8(vld1q_u8((const unsigned char*)ptr))); } \
```

Comment:
> _aligned

This requires that pointer is aligned on `sizeof(_Tpv)` (vector size, 16 bytes). So "bytes"-way is not needed here.

There is same note for `v_store_aligned()` calls.